packages/roosterjs-editor-dom/lib/utils/adjustInsertPosition.ts

```diff
@@ -1,32 +1,30 @@
-import { NodeType, PositionType, QueryScope } from 'roosterjs-editor-types';
-import {
-    changeElementTag,
-    contains,
-    createRange,
-    findClosestElementAncestor,
-    getBlockElementAtNode,
-    getTagOfNode,
-    isNodeEmpty,
-    isPositionAtBeginningOf,
-    isVoidHtmlElement,
-    LinkInlineElement,
-    Position,
-    PositionContentSearcher,
-    queryElements,
-    splitBalancedNodeRange,
-    splitTextNode,
-    toArray,
-    unwrap,
-    VTable,
-    wrap,
-} from 'roosterjs-editor-dom';
+import changeElementTag from './changeElementTag';
+import contains from './contains';
+import createRange from '../selection/createRange';
+import findClosestElementAncestor from './findClosestElementAncestor';
+import getBlockElementAtNode from '../blockElements/getBlockElementAtNode';
+import getTagOfNode from './getTagOfNode';
+import isNodeEmpty from './isNodeEmpty';
+import isPositionAtBeginningOf from '../selection/isPositionAtBeginningOf';
+import isVoidHtmlElement from './isVoidHtmlElement';
+import LinkInlineElement from '../inlineElements/LinkInlineElement';
+import Position from '../selection/Position';
+import PositionContentSearcher from '../contentTraverser/PositionContentSearcher';
+import queryElements from './queryElements';
+import splitTextNode from './splitTextNode';
+import toArray from './toArray';
+import unwrap from './unwrap';
+import VTable from '../table/VTable';
+import wrap from './wrap';
+import { NodePosition, NodeType, PositionType, QueryScope } from 'roosterjs-editor-types';
+import { splitBalancedNodeRange } from './splitParentNode';
 
 const adjustSteps: ((
     root: HTMLElement,
     nodeToInsert: Node,
-    position: Position,
+    position: NodePosition,
```

Comment:
Why use NodePosition instead of Position? I thought NodePosition was only for new instances

Reply:
In most cases, if we can use the interface instead of the class, we should use interface. NodePosition is the interface of Position, although the name is not good, this is for history reason. In next major release we can consider rename them.